unified_planning/test/test_pddl_planner.py

```diff
@@ -138,12 +140,26 @@ def test_robot_loader_adv(self):
             self.assertEqual(len(plan.actions()[3].actual_parameters()), 3)
             self.assertEqual(len(plan.actions()[4].actual_parameters()), 3)
 
+            self.assertIn('Domain parsed\nProblem parsed\nGrounding..\nGrounding Time:', planner_output)
+            self.assertIn('Aibr Preprocessing\n|F|:7\n|X|:0\n|A|:15\n|P|:0\n|E|:0\nH1 Setup Time (msec): ', planner_output)
+            self.assertIn('Setting horizon to:NaN\nHelpful Action Pruning Activated\nRunning WA-STAR\nh(n = s_0)=3.0\n', planner_output)
+            self.assertIn('f(n) = 3.0 (Expanded Nodes: 0, Evaluated States: 0, Time: ', planner_output)
+            self.assertIn('f(n) = 4.0 (Expanded Nodes: 2, Evaluated States: 3, Time: ', planner_output)
+            self.assertIn('f(n) = 5.0 (Expanded Nodes: 5, Evaluated States: 6, Time: ', planner_output)
+            self.assertIn('Problem Solved\n\nFound Plan:\n0.0: (move l1 l2 r1)\n1.0: (load l2 r1 c1)\n2.0: (move l2 l3 r1)', planner_output)
+            self.assertIn('3.0: (unload l3 r1 c1)\n4.0: (move l3 l1 r1)\n\nPlan-Length:5\nMetric (Search):5.0\nPlanning Time (msec):', planner_output)
+            self.assertIn('\nHeuristic Time (msec): ', planner_output)
+            self.assertIn('\nSearch Time (msec): ', planner_output)
+            self.assertIn('\nExpanded Nodes:7\nStates Evaluated:8\n', planner_output)
+            self.assertIn('\nFixed constraint violations during search (zero-crossing):0\nNumber of Dead-Ends detected:0\n', planner_output)
+            self.assertIn('\nNumber of Duplicates detected:8\n', planner_output)
```

Comment:
OK! maybe just grep a line or two otherwise a change in ENHSP would break the test